Dockerized apb-tool not able to perform push

Installing the `apb` CLI locally, via dnf (version `1.1.1.1`), I am able to push to Openshift/ASB. But running into exactly this problem: 
https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/issues/180

However, when I am using the same cluster environment (I am using the `run_latest_build.sh` from the ASB), I am not able to perform a push, like:

```
docker run --rm --privileged -v $(PWD):/mnt:z -v $(HOME)/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $(USER) docker.io/ansibleplaybookbundle/apb-tools:latest push --openshift
```

This does actually result in this problem:

```
Successfully built a2950fdafd98
docker run --rm --privileged -v /home/Matthias/Work/Conferences/DevConf/kafka-apb:/mnt:z -v /home/Matthias/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u 1000 docker.io/ansibleplaybookbundle/apb-tools:latest push --openshift
2017-12-19 17:12:26,836 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x3e7d510>: Failed to establish a new connection: [Errno 111] Connection refused',)': /oapi/v1/namespaces/ansible-service-broker/routes
2017-12-19 17:12:26,837 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x3e7d3d0>: Failed to establish a new connection: [Errno 111] Connection refused',)': /oapi/v1/namespaces/ansible-service-broker/routes
2017-12-19 17:12:26,837 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x3e7d110>: Failed to establish a new connection: [Errno 111] Connection refused',)': /oapi/v1/namespaces/ansible-service-broker/routes
2017-12-19 17:12:26,907 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x3e6eed0>: Failed to establish a new connection: [Errno 111] Connection refused',)': /api/v1/namespaces/default/services/docker-registry
2017-12-19 17:12:26,908 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x3e6e890>: Failed to establish a new connection: [Errno 111] Connection refused',)': /api/v1/namespaces/default/services/docker-registry
2017-12-19 17:12:26,908 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x3e6e750>: Failed to establish a new connection: [Errno 111] Connection refused',)': /api/v1/namespaces/default/services/docker-registry
version: 1.0
name: kafka-apb
description: APB for Apache Kafka and Apache Zookeeper
bindable: False
async: optional
metadata:
  displayName: Apache Kafka Cluster
  imageUrl: "https://svn.apache.org/repos/asf/kafka/site/logos/kafka-logo-no-text.png"
  documentationUrl: "http://kafka.apache.org/documentation"
  providerDisplayName: "Matthias Wessendorf"
plans:
  - name: default
    description: This default plan deploys kafka-apb
    free: True
    metadata: {}
    parameters: []
Exception occurred! HTTPSConnectionPool(host='127.0.0.1', port=8443): Max retries exceeded with url: /api/v1/namespaces/default/services/docker-registry (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x3e7b490>: Failed to establish a new connection: [Errno 111] Connection refused',))
make: *** [Makefile:20: apb_push] Error 1

```

I have a make file that basically embeds the `apb push --openshift1`

--- a/docs/apb_cli.md
+++ b/docs/apb_cli.md
@@ -47,6 +47,12 @@ Pull the container:
 ```bash
 docker pull docker.io/ansibleplaybookbundle/apb-tools
 ```
+
+Create an alias in your `.bashrc` or somewhere else sane for your shell:
+```bash
+alias apb='docker run --rm --privileged --net=host -v $PWD:/mnt -v $HOME/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $UID docker.io/ansibleplaybookbundle/apb-tools'
+```
+
 There are three tags to choose from:
 - **latest**: more stable, less frequent releases
 - **nightly**: following upstream commits, installed from RPM

